packages/react-core/src/index.d.ts

```diff
@@ -5,6 +5,8 @@ export {
   REQUEST_GET_MAX_URL_LENGTH
 } from './utils/requestsUtils';
 
+export { getMonday } from './utils/dateUtils';
```

Comment:
if getMonday is public, then we should add a clearer name to its param: so from `export function getMonday(d) {` --> `export function getMonday(date) {`

Reply:
Totally agree. Updated.